### PR TITLE
Convert web site to use docusaurus

### PR DESCRIPTION
**Pull request by abailly-iohk**
This PR proposes to convert the website generator for cardano-updates to [docusaurus](https://docusaurus.io/), a modern static website generator as advertised by the site. 

The output is visible here: https://abailly-iohk.github.io/cardano-updates/

Pros:
* Default output is visually more appealing and richer than Hakyll (menus, links, automatic tags handling, light/dark mode...)
* Docusaurus is very flexible and extensible with a relatively modest learning curve, it's easy to extend it to include other type of pages generated from elsewhere
* It's actively supported and has a very large community with tons of plugins, including support for i18n 

Cons:
* It's React hence JS/TS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
 
-```bash
-cabal run site -- watch
 ```
+---
+title: DB Sync Team Update
+slug: 2022-10-04-db-sync
+authors: coot
+tags: [db-sync]
+hide_table_of_contents: false
```
Comment (coot) [inline README.md:21]: :smile: 

```suggestion
title: Networking Team Update
slug: 2022-10-04-db-sync
authors: coot
tags: [network]
hide_table_of_contents: false
```

File: blog/2022-08-12-network.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+title: Network Team Update
+slug: 2022-08-12-network
+authors: coot
+tags: [network]
+hide_table_of_contents: false
+---
+
+```mdx-code-block
+import  ReactPlayer from 'react-player'
+```
```
Comment (coot) [inline blog/2022-08-12-network.md:11]: that's cool

File: blog/2022-08-12-network.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  [input-output-hk/ouroboros-network#3922](https://github.com/input-output-hk/ouroboros-network/pull/3922)
+
+* Duncan iterated on his simulation / visualisation.  He also was able to
+  identify and fix a bug in the simulator.  The simulation contains 50 nodes.
+  Dashed lines indicate and established connection, while solid lines indicate
+  a TCP connection with fully open TCP window.
+
+```mdx-code-block
+<ReactPlayer playing controls url='/p2p-relay3.mp4' />
+```
```
Comment (coot) [inline blog/2022-08-12-network.md:46]: I see no video with supported format and MIME type found, but I've seen that before.  Could be something in my browser (although when used a local server it displays the `mp4` file).  Does it work for you?

**Review by coot — APPROVED** [review]
Thanks @abailly, it's awesome.

File: README.md [hunk 4]
```diff
@@ -1,39 +1,31 @@
 # Cardano Development Updates
 
-This repository will host updates from the
-[cardano](https://github.com/input-output-hk/cardano-node) development teams.
-The generated website containing the updates can be found
-[here](https://input-output-hk.github.io/cardano-updates/).
+This repository will host updates from the [cardano](https://github.com/input-output-hk/cardano-node) development teams.
```
Comment (dnadales) [inline README.md:3]: Minor: I'd rather keep the line length to ~80 chars. 

If we wanted to change this anyway maybe we could have made it a separate PR to minimise the diff.

File: README.md [hunk 5]
```diff
@@ -47,3 +39,43 @@ The options include:
 
 Regardless of the workflow chosen, it is important that we strive to keep the git history clean.
 This means never force pushing on main, and usually having one commit per new post.
+
+## Install
```
Comment (dnadales) [inline README.md:43]: Would it make sense to change the title to something like: "Testing the generated content" or something like that?

File: README.md [hunk 6]
```diff
@@ -47,3 +39,43 @@ The options include:
...
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
```
Comment (dnadales) [inline README.md:67]: Aren't the pages automatically deployed?

File: blog/2022-08-12-network.md (hunk 3 above)
Comment (dnadales) [reply blog/2022-08-12-network.md:46]: Hmm, I can't even play it in Chrome :thinking: 

Firefox shows the same error.

File: blog/2022-08-31-consensus.md [hunk 7]
```diff
@@ -1,3 +1,15 @@
+---
+title: Consensus Team Update
+slug: 2022-08-31-consensus
+authors:
+  - name: Damian Nadales
```
Comment (dnadales) [inline blog/2022-08-31-consensus.md:5]: Do we need to set this once in one post and it shows in all the posts? I wonder if we could set this somewhere else. Maybe we need to document this feature in the README as well.

File: blog/2022-09-20-consensus.md [hunk 8]
```diff
@@ -1,3 +1,11 @@
+---
+title: Consensus Team Update
+slug: 2022-09-20-consensus
+authors: dnadales
```
Comment (dnadales) [inline blog/2022-09-20-consensus.md:4]: Do you know how where the mapping `dnadales` to my name and role is defined?

Comment (dnadales) [reply blog/2022-09-20-consensus.md:4]: Nevermind, I was doing a linear scan of the PR. Would it be worth mentioning the `authors` file in the README?

File: docs/intro.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (dnadales) [inline docs/intro.md:1]: Should we delete this?

File: docs/tutorial-basics/_category_.json [hunk 10]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (dnadales) [inline docs/tutorial-basics/_category_.json:1]: Delete?

File: docs/tutorial-basics/congratulations.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (dnadales) [inline docs/tutorial-basics/congratulations.md:1]: Delete?

**Comment by dnadales** [comment]
Thank you so much for doing this Arnaud. It seems we should remove the `docs` folders and probably address some minor remarks. Let us know if you want any of us to take this PR across the finish line.

File: README.md (hunk 1 above)
Comment (abailly-iohk) [reply README.md:21]: Yeah, copy/paste FTW

File: blog/2022-08-12-network.md (hunk 2 above)
Comment (abailly-iohk) [reply blog/2022-08-12-network.md:11]: If it works!

File: blog/2022-08-12-network.md (hunk 3 above)
Comment (abailly-iohk) [reply blog/2022-08-12-network.md:46]: Like I mentioned in #12 there's something fishy going on with checking out the videos

File: README.md (hunk 4 above)
Comment (abailly-iohk) [reply README.md:3]: Good point, I usually don't care but I am fine sticking to that 80 chars limit.

File: README.md (hunk 6 above)
Comment (abailly-iohk) [reply README.md:67]: That's the default README, I will remove

File: blog/2022-08-31-consensus.md (hunk 7 above)
Comment (abailly-iohk) [reply blog/2022-08-31-consensus.md:5]: Good point, there's the `authors.yml` file that allows only given a nickname.

**Comment by coot** [comment]
Closing in favour of #32.

**Pull request closed without merging**